### PR TITLE
chore: Fix typos

### PR DESCRIPTION
**Pull request by NathanBaulch**
Just thought I'd contribute some typo fixes, nothing controversial (hopefully).

Use the following command to get a quick and dirty summary of the specific corrections made:
```shell
git diff HEAD^! --word-diff-regex='\w+' -U0 \
  | grep -E '\[\-.*\-\]\{\+.*\+\}' \
  | sed -r 's/.*\[\-(.*)\-\]\{\+(.*)\+\}.*/\1 \2/' \
  | sort | uniq -c | sort -n
```

FWIW, the top typos are:
* mmaped (29)
* hashicups (12)
* lablel (8)
* formated (6)
* timeserie (6)
* additionnal (2)
* mmap'd (2)
* mutuall (2)
* sclaing (2)
* exemple (2)
* compactible (2)
* homogenous (2)

### CODE REVIEW

File: discovery/ovhcloud/vps.go
Comment (juliusv) [inline discovery/ovhcloud/vps.go:40]: As this one touches external JSON (un)marshaling, I just want to confirm with @Mama59 / @Arnoways that this is ok? Just in case the external OVH API actually has the field misspelled and we are relying on that or something.

File: tsdb/agent/db.go
Comment (juliusv) [inline tsdb/agent/db.go:338]: I think the other two `{Default}StripSize` mentions here should be `{Default}StripeSize` too.

File: tsdb/db_test.go
Comment (juliusv) [inline tsdb/db_test.go:7471]: "compactible" seems correct: https://www.dictionary.com/browse/compactible

**Review by juliusv — CHANGES_REQUESTED** [review]
Thanks! A few comments:

File: discovery/nomad/nomad_test.go
Comment (juliusv) [reply discovery/nomad/nomad_test.go:75]: Or rather, Arve noticed it already.

**Comment by juliusv** [comment]
Besides the comments, the Go code will need to be `gofmt`-ed again. Currently the CI tests are failing because the formatting is off now.

File: discovery/ovhcloud/vps.go
Comment (Arnoways) [reply discovery/ovhcloud/vps.go:40]: The API is actually misspelled, it should stay this way

**Comment by aknuds1** [comment]
Awaiting the fixes according to Julius' review :)

File: discovery/nomad/nomad_test.go
Comment (NathanBaulch) [reply discovery/nomad/nomad_test.go]: Oops, reverted!

File: discovery/nomad/nomad_test.go
Comment (NathanBaulch) [reply discovery/nomad/nomad_test.go]: Oops, reverted!

File: tsdb/agent/db.go
Comment (NathanBaulch) [reply tsdb/agent/db.go]: Done.

File: tsdb/db_test.go
Comment (NathanBaulch) [reply tsdb/db_test.go:7471]: I went with "-able" because there were already 30 instances of "compactable" in the project, including 6 in that same file.

File: discovery/ovhcloud/vps.go
Comment (NathanBaulch) [reply discovery/ovhcloud/vps.go:40]: "it should stay this way" meaning it should stay misspelled (as it was), or it should stay corrected (as it currently is in this PR branch)?

File: tsdb/db_test.go
Comment (aknuds1) [reply tsdb/db_test.go:7471]: Is there any point in changing it though, considering it's correct? I personally think it should be left be.

File: discovery/ovhcloud/vps.go
Comment (aknuds1) [reply discovery/ovhcloud/vps.go:40]: I think it means the PR shouldn't change this, since it reflects the API.

Comment (Arnoways) [reply discovery/ovhcloud/vps.go]: > I think it means the PR shouldn't change this, since it reflects the API.

Correct, the PR should not change this.

**Review by juliusv — APPROVED** [review]
Thanks, looks good now I think! :+1: (I don't mind the compactable/compactible spelling either way.)

**Comment by juliusv** [comment]
@aknuds1 it's still saying it's blocked on you requesting changes, but hopefully should be good now, feel free to approve + merge :)

**Review by aknuds1 — APPROVED** [review]
LGTM, thanks!

**Comment by aknuds1** [comment]
I'll merge after looking over a last time. Busy w/ PromCon preparations right now though :)

**Pull request merged**
